pset-functions.js

```diff
@@ -30,6 +30,25 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
+function blackJack (){
+  if(playerCardScore > dealerCardScore && playerCardScore <= 21){
+        return `Player wins with a score of ${playerCardScore}`
+      }
+      else if(dealerCardScore > playerCardScore && dealerCardScore <= 21) {
+        return `The dealer wins with a score of ${dealerCardScore}`
+      }
+      else {
+        return `Bust! Player loses with a score of ${playerCardScore}, and the dealer loses with a score of ${dealerCardScore}`
+      }
+
+}
+
+const playerCardScore = 22
+const dealerCardScore = 22
+
+
+console.log(blackJack())
+
```

Comment:
@Nice Job with this solution here. Your `blackJack` function looks really good, I see you're utilizing conditionals to do your checks to see what is the outcome of the game. One suggestion when you call the function you would want to pass the variables `playerCardScore` and `dealerCardScore` as parameters like this `blackjack(playerCardScore,dealerCardScore)` then your main function would look like this ex. `function blackJack(variable1,variable2)`. 
take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L30)